db/multi_cf_iterator.h

```diff
@@ -72,18 +73,65 @@ class MultiCfIterator : public Iterator {
     const Comparator* comparator_;
   };
 
+  class MultiCfMaxHeapItemComparator {
```

Comment:
It would be possible to avoid this copy-paste class with a template. Not a big deal.